From 085bab53daa7897ca481598af769900686c8b41f Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Date: Mon, 17 Jun 2013 13:47:26 +0200
Subject: [PATCH 0651/1083] ARM: shmobile: r8a7790: Add LVDS support

Signed-off-by: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
(cherry picked from commit b862d97c90a346376d47f8d768008849a0cc768c)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit f3e462908ce26adec3b734ca5517de12b7e00840)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7790.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-shmobile/setup-r8a7790.c b/arch/arm/mach-shmobile/setup-r8a7790.c
index 05d41d7..6f0bd4f 100644
--- a/arch/arm/mach-shmobile/setup-r8a7790.c
+++ b/arch/arm/mach-shmobile/setup-r8a7790.c
@@ -98,6 +98,8 @@ void __init r8a7790_pinmux_init(void)
 /* DU */
 static const struct resource du_resources[] = {
 	DEFINE_RES_MEM(0xfeb00000, 0x70000),
+	DEFINE_RES_MEM_NAMED(0xfeb90000, 0x1c, "lvds.0"),
+	DEFINE_RES_MEM_NAMED(0xfeb94000, 0x1c, "lvds.1"),
 	DEFINE_RES_IRQ(gic_spi(256)),
 	DEFINE_RES_IRQ(gic_spi(268)),
 	DEFINE_RES_IRQ(gic_spi(269)),
-- 
1.8.3.2

